### PR TITLE
Blog for Automating via Argo

### LINKED ISSUE #12: Automation via Argo blog
As a developer, we want to create summaries or blogs from the first batch of our existing consumable items so that they can be used as an outward facing blog.
aicoe-aiops/ai-enablement-initiative#4

Tasks to be done :
*  Convert to asciidoc 
*  Add to docs folder in the repo and link the video to the doc 
*  Modify the header, title, author, tags etc

**Acceptance Criteria** : Created blogs for projects, get them reviewed and converted them into ascii docs and added them to individual git repository.

### PR DESCRIPTION
**Pull request by aakankshaduggal**
Fixes #12

### CODE REVIEW

File: docs/automating-via-argo.adoc [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+
+As you can see, extending the workflow with a tasks to run a new analysis is fairly straightforward, all it takes are those 4 lines.
+
+If you drill down into our Argo setup you may discover more fine details, like the rendered notebook collection mechanisms or dataset collection task graph.
+
+## Result
+
+We piloted this automation process on the https://github.com/aicoe-aiops/openshift-sme-mailing-list-analysis[OpenShift SME mailing list analysis] project. The automation result - a workflow, when executed, looks like this:
+
+image:../assets/images/workflow_diagram.png[Workflow diagram]
```
[the remark this answers is not in the document]
Comment (aakankshaduggal) [reply docs/automating-via-argo.adoc:212]: @tumido I am using the images from already existing [folder](https://github.com/aicoe-aiops/data-science-workflows/tree/master/assets/images) in the repo.

[the remark this answers is not in the document]
Comment (tumido) [reply docs/automating-via-argo.adoc:212]: Ahh, I see.. Sorry, I've confused the repo this PR is open against. :smile:  So.. can you please refresh for me, why do we need to have the document twice in this repo then? Maybe you want to remove the original markdown file then? :slightly_smiling_face:

[the remark this answers is not in the document]
Comment (aakankshaduggal) [reply docs/automating-via-argo.adoc:212]: As a part of AI-enablement initiative, we are trying to publish our blogs to [help.dh](help.datahub.redhat.com). 
https://github.com/aicoe-aiops/ai-enablement-initiative/issues/4

In order to do so, we need the blog in the  ascii doc format and in `docs` folder of each repository. 

Also, regarding the markdown file, I guess it's safe to remove :+1:

[the remark this answers is not in the document]
Comment (tumido) [reply docs/automating-via-argo.adoc:212]: Let's remove it then, please :+1:

[the remark this answers is not in the document]
Comment (aakankshaduggal) [reply docs/automating-via-argo.adoc:212]: Done :+1:

**Comment by aakankshaduggal** [comment]
ping @MichaelClifford

**Comment by aakankshaduggal** [comment]
@MichaelClifford Could you please merge this too ?

**Pull request merged**